lib/output/markdown.js

```diff
@@ -12,18 +116,23 @@ var extend = require('extend'),
  *
  * @param {Array<Object>} comments parsed comments
  * @param {Object} opts Options that can customize the output
- * @param {String} [opts.template='../../share/markdown.hbs'] Path to a Handlebars template file that
+ * @param {string} [opts.template='../../share/markdown.hbs'] Path to a Handlebars template file that
  * takes the place of the default.
  * @param {Function} callback called with array of results as vinyl-fs objects
  * @name markdown
  * @return {undefined} calls callback
  */
 module.exports = function (comments, opts, callback) {
-  var options = extend({}, {
-    theme: 'documentation-theme-default'
-  }, opts);
-  var themeModule = resolveTheme(options.theme);
-  var template = getTemplate(Handlebars, themeModule, 'markdown.hbs');
-  helpers(Handlebars);
-  return callback(null, comments.map(template).join(''));
+
+  opts = opts || {};
+
+  if (opts.toc || true) {
+    mdast.use(toc);
+  }
+
+  var ast = u('root', comments.map(function (comment) {
+    return generate(1, comment);
+  }));
+
+  return callback(null, mdast.stringify(ast));
```

Comment:
Could we export a function that returns the unstringified ast?  We could leave the default export as is, but when we do https://github.com/documentationjs/documentation-readme/issues/4, it would be nice to grab the ast rather than stringifying here only to reparse in `documentation-readme`